springboard_iogt_ug/views.py

```diff
@@ -0,0 +1,30 @@
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound, HTTPNotFound
+
+from springboard.utils import ga_context
+from springboard_iogt.views import IoGTViews, PERSONA_COOKIE_NAME, ONE_YEAR
+from springboard_iogt.utils import (
+    get_redirect_url)
+
+PERSONAE = {'YOUTH_BOY', 'YOUTH_GIRL', 'FATHER', 'MOTHER'}
+
+
+class UGIogtViews(IoGTViews):
+    @ga_context(lambda context: {'dt': 'Selected Persona', })
+    @view_config(route_name='select_persona')
+    def select_persona(self):
```

Comment:
Do method decorators work on class methods? I read something last night that said they don't. How are these decorators?
https://docs.djangoproject.com/en/1.8/topics/class-based-views/intro/#id2